buildslave.rst

```diff
@@ -196,7 +196,7 @@ Latent slaves
 ^^^^^^^^^^^^^
 
 We also support running `latent buildslaves
-<http://docs.buildbot.net/current/manual/cfg-buildslaves.html#latent-buildslaves>`_
+<http://docs.buildbot.net/0.8.14/manual/cfg-buildslaves.html#latent-buildslaves>`_
```

Comment:
We're now on current buildbot; this page has also been updated in the meanwhile.